tool/gravity/cli/register.go

```diff
@@ -205,8 +205,12 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.UpdateTriggerCmd.Manual = g.UpdateTriggerCmd.Flag("manual", "Manual operation. Do not trigger automatic update.").Short('m').Bool()
 	g.UpdateTriggerCmd.SkipVersionCheck = g.UpdateTriggerCmd.Flag("skip-version-check", "Bypass version compatibility check.").Hidden().Bool()
 	g.UpdateTriggerCmd.Force = g.UpdateTriggerCmd.Flag("force", "Force update operation even if some nodes have active warnings.").Bool()
+	g.UpdateTriggerCmd.SkipWorkers = g.UpdateTriggerCmd.Flag("skip-workers", "Skip upgrade of worker nodes").Hidden().Bool()
+	g.UpdateTriggerCmd.ParallelWorkers = g.UpdateTriggerCmd.Flag("parallel-workers", "Number of worker nodes to upgrade in parallel (Default: 1)").Int()
```

Comment:
Where is the default set to 1?

Reply:
The default for the variable isn't really set to 1, as you noted above, the default behaviour is serial execution, which is basically the same as parallel execution where only 1 is allowed at a time.

Reply:
ended up tweaking this in 22eb5a3b292090bd33fb5ba4a7a9a371132283f2

Reply:
Apparently I brain farted and kingpin doesn't really reflect an unset value, so I had to tweak this to only detect negative numbers in aaf1809dc1b2b16438f8170ccf4b2c7e2831bf94 and otherwise keep the logic to do serial invocation for a 0 or 1.